src/legacy/ui/public/timefilter/index.ts

```diff
@@ -18,20 +18,20 @@
  */
 
 import uiRoutes from 'ui/routes';
-import { TimefilterContract, TimeHistoryContract } from '../../../core_plugins/data/public';
+import { npStart } from 'ui/new_platform';
+import { TimefilterContract, TimeHistoryContract } from '../../../../plugins/data/public';
 import { registerTimefilterWithGlobalState } from './setup_router';
-import { start as data } from '../../../core_plugins/data/public/legacy';
 
 export {
   getTime,
   InputTimeRange,
   TimeHistoryContract,
   TimefilterContract,
-} from '../../../core_plugins/data/public';
+} from '../../../../plugins/data/public';
 export type Timefilter = TimefilterContract;
 export type TimeHistory = TimeHistoryContract;
-export const timeHistory = data.timefilter.history;
-export const timefilter = data.timefilter.timefilter;
+export const timeHistory = npStart.plugins.data.query.timefilter.history;
+export const timefilter = npStart.plugins.data.query.timefilter.timefilter;
```

Comment:
Is the double `timefilter` key &mdash; `.timefilter.timefilter` &mdash; temporary, or we are planning to keep that?

Reply:
ATM it's just backwards compatible.
Though I'm not a big fan as well.
I'm open to making a follow up PR updating the structure.